src/lib/vm/vm.js

```diff
@@ -617,7 +617,7 @@ class VmStack {
       return <Widget {...attributes} />;
     } else if (element === "CommitButton") {
       return (
-        <CommitButton {...attributes} widgetSrc={this.vm.widgetSrc}>
+        <CommitButton {...attributes} widgetSrc={this.vm.widgetSrc} chainId={this.vm.widgetConfigs.findLast(config => config && config.chainId)?.chainId}>
```

Comment:
Better to refactor it. Since it uses injected `near` from `Widget`. it already should have the `networkId`

Reply:
Done ✅